src/components/SlateEditor/plugins/file/Filelist.jsx

```diff
@@ -254,15 +247,20 @@ class Filelist extends React.Component {
             onClose={this.onCloseFileUploader}
             showFileUploader={showFileUploader}
           />
+          {children}
         </StyledSection>
       </Fragment>
     );
   }
 }
 
 Filelist.propTypes = {
+  attributes: PropTypes.shape({
+    'data-key': PropTypes.string.isRequired,
+  }),
   editor: EditorShape,
   node: PropTypes.any,
```

Comment:
prop `element` erstatter `node`. Siden krasjer nå når man skriver i input-feltet for endring av filnavn. Skjer i funksjonen onUpdateFileName hvor `node` fortsatt brukes